Management_System/DAL/Repositories/Interfaces/IBoardRepository.cs

```diff
@@ -0,0 +1,13 @@
+﻿using DAL.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DAL.Repositories.Interfaces;
+public interface IBoardRepository : IRepository<BoardEntity>
```

Comment:
Please let do code style in one way.
Do we add blank line after namespace or not?

Fix all updated files please